### PR TITLE
feat: implement `voice_channel_effect` event

### PR DESCRIPTION
**Pull request by shiftinv**
## Summary

https://github.com/discord/discord-api-docs/pull/6025

## Checklist

- [x] If code changes were made, then they have been tested
    - [x] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `pdm lint`
    - [x] I have type-checked the code by running `pdm pyright`
- [ ] This PR fixes an issue
- [x] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: disnake/raw_models.py [hunk 1]
```diff
@@ -422,3 +426,77 @@ class RawGuildMemberRemoveEvent(_RawReprMixin):
     def __init__(self, user: Union[User, Member], guild_id: int) -> None:
         self.user: Union[User, Member] = user
         self.guild_id: int = guild_id
+
+
+class VoiceChannelEffect(_RawReprMixin):
```
Comment (Victorsitou) [inline disnake/raw_models.py:431]: How about moving this to `voice_client.py`?

**Comment by onerandomusername** [comment]
@shiftinv would you please resolve conflicts?

**Comment by onerandomusername** [comment]
Do we need to wait on the docs PR for this? I'm not quite sure if they plan to ever merge it or nto

**Comment by shiftinv** [comment]
Probably, yea. It's not that important of a feature anyway, so personally I'm fine with waiting a little longer.

**Pull request merged**